docs/contracts/contracts/LSP7DigitalAsset/LSP7DigitalAsset.md

```diff
@@ -1216,3 +1216,190 @@ reverts when trying to authorize or revoke the token's owner as an operator.
 [`LSP10Constants.sol`]: https://github.com/lukso-network/lsp-smart-contracts/tree/main/contracts/LSP10ReceivedVaults/LSP10Constants.sol
 
 <!-- prettier-ignore-end -->
+
+<!-- GLOBAL LINKS -->
+
+<!-- prettier-ignore-start -->
```

Comment:
I think you don't need to put again a `<!-- prettier-ignore-start -->`, since there is already one at the top.

<img width="1048" alt="image" src="https://github.com/lukso-network/docs/assets/31145285/d3618af9-1e8d-487e-acb8-12ec58807797">

You can remove simply this one and keep the closing one at the bottom.

<img width="403" alt="image" src="https://github.com/lukso-network/docs/assets/31145285/462b9502-4aab-42e5-9a46-c19b463ea926">